src/components/common/Gluestack/core/Icons/Icons/Favourite.tsx

```diff
@@ -0,0 +1,24 @@
+import { createIcon } from "@gluestack-ui/icon";
+import React from "react";
+import { Path } from "react-native-svg";
+import { Root } from "../styled-components";
+
+const FavouriteIcon: any = createIcon({
```

Comment:
That's one of the Gluestack generated component so maybe the dev was a Brit? Most of those icons can probably be deleted since I don't think we're likely to use them.